bin/dotd

```diff
@@ -205,6 +205,25 @@ def content_push(server_name, environment_name)
   end
 end
 
+# Asks if the robo-DTT is green, if the DTT message should be reset, and if Honeybadger issues
+# should be shown.
+def dtt_routine
```

Comment:
For what it's worth, while `routine` and `subroutine` are both correct terms for a 'callable unit', I think `subroutine` is more common and less ambiguous.  At least, it got the title position [on Wikipedia](https://en.wikipedia.org/wiki/Subroutine).  Alternatively, I think `method` is the correct term in ruby - and all of the above are probably considered rendant as part of a method name, esp. in the Ruby idiom.